<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoPoster</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="discordToken">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Discord token</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="randomTimeCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid rgb(226, 193, 255); 
color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Ранд. промежуток</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="randomDelayMAX">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>Макс. промежуток (минут)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>Промежуток (минут)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deleteMessageCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid rgb(226, 193, 255); 
color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Удалять сообщения</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="randomDelayMIN">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>Мин. промежуток (минут)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="warpName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Название варпа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="channelID">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Channel ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="messageContent">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>220</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Сообщение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid rgb(214, 198, 255);
background-color: rgb(214, 198, 255);</string>
    </property>
    <property name="text">
     <string>СТАРТ</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
